libs/client-api/src/http_view.rs

```diff
@@ -21,6 +21,23 @@ impl Client {
     AppResponse::<Page>::from_response(resp).await?.into_data()
   }
 
+  pub async fn move_workspace_page_view_to_trash(
+    &self,
+    workspace_id: Uuid,
+    view_id: String,
+  ) -> Result<(), AppResponseError> {
+    let url = format!(
+      "{}/api/workspace/{}/page-view/{}/move-to-trash",
+      self.base_url, workspace_id, view_id
+    );
+    let resp = self
+      .http_client_with_auth(Method::POST, &url)
```

Comment:
maybe we should use `Method::DELETE` instead? then you dont need the `move-to-trash` in the path